project/plugins.sbt

```diff
@@ -25,7 +25,7 @@ libraryDependencies += "com.puppycrawl.tools" % "checkstyle" % "9.3"
 // checkstyle uses guava 31.0.1-jre.
 libraryDependencies += "com.google.guava" % "guava" % "31.0.1-jre"
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "1.2.0")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "2.0.0")
```

Comment:
Can you help to check whether the contents of the `connect` assembly jar are consistent before and after the pr?

Reply:
They are consistent

> When 1.2.0:
<img width="930" alt="image" src="https://user-images.githubusercontent.com/15246973/205833991-afd564b1-7441-42ae-90d9-37f1fffa782d.png">

> After 2.0.0:
<img width="933" alt="image" src="https://user-images.githubusercontent.com/15246973/205834718-3424487f-d66b-4e17-b331-6be20b577e81.png">